src/gui/guiFormSpecMenu.cpp

```diff
@@ -2084,31 +2084,89 @@ void GUIFormSpecMenu::parseBox(parserData* data, const std::string &element)
 			geom.Y = stof(v_geom[1]) * spacing.Y;
 		}
 
+		FieldSpec spec(
+			"",
+			L"",
+			L"",
+			258 + m_fields.size(),
+			-2
+		);
+		spec.ftype = f_Box;
+
+		auto style = getStyleForElement("box", spec.fname);
+
 		video::SColor tmp_color;
+		std::vector<video::SColor> colors;
+		std::vector<video::SColor> bordercolors;
+		std::vector<s32> borderwidths;
+
+		// Colors
+		if (parseColorString(parts[2], tmp_color, true, 0x8C))
+			colors = {tmp_color, tmp_color, tmp_color, tmp_color};
+		else {
```

Comment:
https://www.kernel.org/doc/html/latest/process/coding-style.html#placing-braces-and-spaces

Please put brackets around the first code block as well. Same for L2117, L2124, ...